stable/pod-gpu-metrics-exporter/templates/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "pod-gpu-metrics-exporter.serviceAccountName" . }}
+  labels:
+{{ include "pod-gpu-metrics-exporter.labels" . | indent 4 }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
+  - patch
+  - update
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: {{ include "pod-gpu-metrics-exporter.fullname" . }}
+  labels:
+{{ include "pod-gpu-metrics-exporter.labels" . | indent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
```

Comment:
```suggestion
  kind: ClusterRole
```

Reply:
just a typo.... definitely must be `Role` in our case